public/app/features/logs/components/LogRowMessage.tsx

```diff
@@ -117,9 +119,19 @@ export class LogRowMessage extends PureComponent<Props> {
               fill="text"
               size="md"
               getText={this.getLogText}
-              tooltip="Copy"
+              tooltip="Copy logline to clipboard"
```

Comment:
```suggestion
              tooltip="Copy log line to clipboard"
```

Reply:
We use log line in documentation 
<img width="875" alt="image" src="https://github.com/grafana/grafana/assets/30407135/d34d69fb-4387-441f-8146-204cb93e9e27">

Reply:
Actually, maybe we could remove `log line` from everywhere and just have `Show context`, `Copy to clipboard`, `Copy shortlink to clipboard`

Reply:
Yea I like that more.  Fixed in ad6baafe6937c8abeccacf29d3c2c19a37baa94f